src/MagnumPlugins/DdsImporter/Test/DdsImporterTest.cpp

```diff
@@ -241,6 +254,153 @@ void DdsImporterTest::dxt5() {
             TestSuite::Compare::Container);
 }
 
+void DdsImporterTest::dxt10formats2d() {
+    static struct TestFile {
```

Comment:
I fear that the `static` keyword implies some unnecessary code generation (mutexes for thread safety). I'm completely okay with this being a `constexpr` anonymous struct in an anonymous namespace *outside* of a function. Details how that might look like [again in the docs](http://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#details) (sorry) ;)